mllib/src/main/scala/org/apache/spark/mllib/regression/Lasso.scala

```diff
@@ -37,6 +40,13 @@ class LassoModel (
   extends GeneralizedLinearModel(weights, intercept)
   with RegressionModel with Serializable with Saveable with PMMLExportable {
 
+  override protected def predictPointWithProbability(
+                                       dataMatrix: Vector,
+                                       weightMatrix: Vector,
+                                       intercept: Double) = {
+    throw new Exception("Not implemented for LassoModel")
```

Comment:
Without commenting on the suitability of the PR -- I can at least say that `throw new Exception` is never a great idea. Use `UnsupportedOperationException` here, for example.